deltatech_product_tracking/models/product.py

```diff
@@ -4,6 +4,19 @@
 from odoo import api, models
 
 
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    def write(self, vals):
+        res = super(ProductTemplate, self).write(vals)
```

Comment:
Ar trebui sa filtram template si product inainte si sa rulam doar pentru alea, nu pentru toate:

    def write(self, vals):
        templates_without_lot = self.filtered(lambda p: p.tracking == "none")
        res = super(ProductTemplate, self).write(vals)
        if templates_without_lot and vals.get("tracking", "none") == "lot":
            templates_without_lot.mapped("product_variant_ids").fill_lot_in_stock_docs()
        return res